src/ray/core_worker/test/direct_actor_transport_test.cc

```diff
@@ -904,30 +905,32 @@ TEST_F(TaskReceiverTest, TestNewTaskFromDifferentWorker) {
   {
     auto request =
         CreatePushTaskRequestHelper(actor_id, 0, worker_id, caller_id, curr_timestamp);
-    rpc::PushTaskReply reply;
```

Comment:
1. If we don't use a pointer, the stack will have been freed by the time the executor tries to access it.
2. If I use a smart pointer, I need to update the function signature.

Reply:
chatted with @dentiny: use scope exit (abseil cleanup) or shared pointer